src/api/WordsAPI.ts

```diff
@@ -80,6 +82,11 @@ export type SearchResult = ReturnType<typeof getSearchResults> extends Promise<
 export type SearchTerm = SearchResult["items"] extends (infer U)[] ? U : never;
 
 export const useSearch = (word: string | undefined) => {
+  //removes all trailing whitespaces
+  word = word?.replace(/[ /\t]+$/, "");
+  //removes all leading whitespaces
+  word = word?.replace(/^[ /\t]+/, "");
```

Comment:
You know, there is this function called `trim` :-)
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/Trim